### PR TITLE
Add documentation for sideloading

### PR DESCRIPTION
**Pull request by jenssegers**
Documenting our sideloading feature.

### CODE REVIEW

File: src/sideloading.apib [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ]
+  }
+}
+```
+
+Note that the sideloaded relationships are indexed based on their object type. Use the `type` property from your original relationship object to search for the sideloaded relationship in the `included` section.
+
+Multiple relationships can be sideloaded at the same time by passing them as a comma-separated value such as `include=lead.company,responsible_user`.
+
+Sideloading is supported on every endpoint which returns objects.
```
Comment (WouterSioen) [inline src/sideloading.apib]: PP: set up your editor to always add an end-of-file character.

**Review by WouterSioen — DISMISSED** [review]

File: src/sideloading.apib (hunk 1 above)
Comment (jenssegers) [reply src/sideloading.apib]: Used Grammarly 😅

Comment (WouterSioen) [reply src/sideloading.apib]: You did not commit from grammarly, did you? 😉

**Review by WouterSioen — APPROVED** [review]

**Review by psrpinto — APPROVED** [review]
🚢

**Review by Jdww — APPROVED** [review]

**Review by marijnvandevoorde — APPROVED** [review]
:shipit: jenske!

**Comment by stivni** [comment]
Before shipping, I'll ask Niels to gather some feedback from trusted partners about the response format...

**Comment by nielsvandenb** [comment]
Comment from Niko Nelissen of Belndr.io: Cool, works fine. I tested the deal.contact. I noticed you get less information than in the contacts.info, example custom fields are not returned, is this supposed to be like this? Nice feature.

**Review by adognameddog — APPROVED** [review]

**Review by dcsg — APPROVED** [review]

**Review by steftrenson — APPROVED** [review]

**Pull request merged**